### DISCUSSION TITLE
Preserve html comments after build

### DISCUSSION THREAD

**Discussion by S-kor-IBM** [vuepress/core#1022] [Q&A] [answered]
Hello, 
We are trying to add some invisible copyright information into every page, so, added something like that to .md files:
```
<!-- 
copyright
-->
```
but "build" to static pages removes all that comments. Is there a way to preserve them? Any other way to accomplish the main goal?
Thank you.

**Comment by meteorlxy** [comment, marked as answer]
You can set compilerOptions via bundler's options

- https://v2.vuepress.vuejs.org/reference/bundler/vite.html#vuepluginoptions
- https://vuejs.org/api/options-rendering.html#compileroptions
